src/source.ts

```diff
@@ -136,6 +136,21 @@ export interface SourceConfig extends NativeInstanceConfig {
   analyticsSourceMetadata?: SourceMetadata;
 }
 
+/**
+ * The remote playback config for a source.
```

Comment:
Aiii
```suggestion
 * The remote control config for a source.
```

Reply:
Looks like mass replace didn't work out 🤦 

Fixed in da16514